bot/code_review_bot/tasks/docupload.py

```diff
@@ -77,8 +77,18 @@ def build_notice(self, artifacts, revision):
         if trees is None:
             logger.warn("Missing trees.json")
 
-        doc_files = [file for file in revision.files if "docs" in file]
+        doc_files = [
+            file
+            for file in revision.files
+            if "docs" in file and file.endswith((".rst", ".md"))
```

Comment:
Could you add a comment mentioning that we should keep the list up to date with https://searchfox.org/mozilla-central/rev/2d24d893669ad0fe8d76b0427b25369d35fcc19b/docs/conf.py#79 ?